app/furniture/marketplace/README.md

```diff
@@ -1,5 +1,18 @@
 # Marketplace
 
+## Human/Computer/Human Interaction Design
+
+### Shopper Purchases Products
+
+<!-- it would be cool if someone could Mermaid this -->
```

Comment:
wow [Mermaid](https://mermaid.js.org/) looks sick/cool! **- TODO @KellyAH  🧜‍♀️ it!**

Reply:
It *is* cool! I am just not fluent in it (yet!)

Reply:
was there a specific type of diagram you hoping for? Sequence, flow, state, user journey or something else? see [Sample Diagrams](https://mermaid.live/edit#pako:eNplULsKwzAM_BWhOUspXbw20MlT1iwiFo3bxgqOTAih_17nBS3VJO5Od5JmbMQxGnxIioGnOkAu9fpisBOMEp8-3MHRzgzcqJcANwGVld5wAEtPBmUycDFg-YCzMPWDko-DgfM3U27zBk4LWsCV9C-jle7byckYDq-flMrrSu4RWGDHsSPv8mXzIqpRW85eaHLrKK-NdXhnHSWVagoNGo2JC0y9I-XS0z1Sd4DsvEq026fWh70_H4Fhow)

Reply:
Here's a rough draft diagram @zspencer 
Is this okay to digitize w/ 🧜‍♀️  or should I "go back to the drawing board"? 😉 

![shopping checkout flow draft](https://user-images.githubusercontent.com/16140873/213841324-4df13266-79d8-46e2-9446-4d1866ab1601.jpg)

Reply:
Coded in 🧜‍♀️  in https://github.com/zinc-collective/convene/pull/1064